src/senaite/core/tests/doctests/AnalysisTurnaroundTimeForZeroValues.rst

```diff
@@ -0,0 +1,196 @@
+    Analysis Turnaround Time For Zero Values
+------------------------
+
+Running this test from the buildout directory::
+
+    bin/test test_textual_doctests -t AnalysisTurnaroundTimeForZeroValues
+
+
+Test Setup
+..........
+
+Needed Imports:
+
+    >>> import re
+    >>> from AccessControl.PermissionRole import rolesForPermissionOn
+    >>> from bika.lims import api
+    >>> from bika.lims.api.analysis import get_formatted_interval
+    >>> from bika.lims.api.analysis import is_out_of_range
+    >>> from bika.lims.content.analysisrequest import AnalysisRequest
+    >>> from bika.lims.utils.analysisrequest import create_analysisrequest
+    >>> from bika.lims.utils import tmpID
+    >>> from bika.lims.workflow import doActionFor
+    >>> from bika.lims.workflow import getCurrentState
+    >>> from bika.lims.workflow import getAllowedTransitions
+    >>> from DateTime import DateTime
+    >>> from datetime import timedelta
+    >>> from plone.app.testing import TEST_USER_ID
+    >>> from plone.app.testing import TEST_USER_PASSWORD
+    >>> from plone.app.testing import setRoles
+    >>> from Products.ATContentTypes.utils import DT2dt, dt2DT
+
+Functional Helpers:
+
+    >>> def start_server():
+    ...     from Testing.ZopeTestCase.utils import startZServer
+    ...     ip, port = startZServer()
+    ...     return "http://{}:{}/{}".format(ip, port, portal.id)
+
+    >>> def change_receive_date(ar, days):
```

Comment:
Please remove this function. Not used anywhere

Reply:
This function is called later, so I didn't remove. If you still want it removed, please let me know.